modules/custom/az_news/az_news.module

```diff
@@ -16,3 +23,47 @@ function az_news_theme($existing, $type, $theme, $path) {
     ],
   ];
 }
+
+/**
+ * Implements hook_entity_extra_field_info().
+ */
+function az_news_entity_extra_field_info() {
+  $extra = [];
+
+  // Check to see if our content type exists.
+  $bundle = NodeType::load('az_news');
+  if ($bundle) {
+    // Link title pseudo field.
+    $extra['node'][$bundle->id()]['display']['az_news_read_more'] = [
+      'label' => t('Read more pseudo link.'),
+      'description' => "This is a pseudo field from az_news",
+      'weight' => 50,
+      'visible' => FALSE,
+    ];
+  }
+
+  return $extra;
+}
+
+/**
+ * Implements hook_ENTITY_TYPE_view().
+ */
+function az_news_node_view(array &$build, EntityInterface $entity, EntityViewDisplayInterface $display, $view_mode) {
+
+  if ($entity instanceof FieldableEntityInterface && $entity->hasField('field_az_link')) {
+    $bubbleable_metadata = new BubbleableMetadata();
+    $token_service = Drupal::token();
+    // Read more pseudo field.
+    if ($display->getComponent('az_news_read_more')) {
+      $build['az_news_read_more'][] = [
+        '#type' => 'markup',
+        '#markup' => Html::escape($token_service
```

Comment:
Nitpick: we should probably be using `Drupal::service()` to get the token service and I don't think the `$token_service` variable is really needed.
```suggestion
    // Read more pseudo field.
    if ($display->getComponent('az_news_read_more')) {
      $build['az_news_read_more'][] = [
        '#type' => 'markup',
        '#markup' => Html::escape(\Drupal::service('token')
```